### PR TITLE
Add new testsuites for installing the official SUSE WSL from the MS Store

### PR DESCRIPTION
**Pull request by pablo-herranz**
Adding new testsuites for installing WSL images directly from the MS Store.

### CODE REVIEW

**Review by grisu48 — APPROVED** [review]
LGTM! However should be merged only after https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15107, right?

File: job_groups/opensuse_leap_15.4_wsl.yaml [hunk 1]
```diff
@@ -49,3 +50,16 @@ scenarios:
             QEMUMACHINE: 'q35,accel=whpx'
             QEMUCPUS: '2'
             WORKER_CLASS: 'openqaworker7'
+      - wsl-install-MSStore-Leap15.4:
```
Comment (jlausuch) [inline job_groups/opensuse_leap_15.4_wsl.yaml:53]: I have added the job to development, you can now remove this part and add it later.

File: job_groups/opensuse_tumbleweed.yaml [hunk 2]
```diff
@@ -1213,6 +1213,7 @@ scenarios:
             VNC_STALL_THRESHOLD: '10'
             VNC_TYPING_LIMIT: '15'
             QEMU_ENABLE_SMBD: '1'
+            INSTALL_FROM: 'build'
```
[the remark this answers is not in the document]
Comment (DimStar77) [reply job_groups/opensuse_tumbleweed.yaml:1216]: > 

Scratch the error - I just see the wsl_settings variable in use... too few coffee in blood :)

**Review by jlausuch — APPROVED** [review]

**Comment by jlausuch** [comment]
When we have a couple of successful runs of the new test in TW Development group, feel free to move it to main TW group :)

**Pull request merged**